exercises/prs.md

```diff
@@ -10,7 +10,7 @@ See [issues.md](issues.md) for how to prepare for this exercise.  If that prep h
 
 This exercise requires working knowledge of creating Github pull requests.  If any students at the table have never done this before, their neighbors should help them.
 
-1. Two students at the table (3 for 10-person tables) who have the status "reviewer" should each create a PR. Check the Owners file for status.
+1. Two students at the table (3 for 10-person tables) who are NOT in the OWNERS file as Reviewer or Approver should each create a PR. Check the Owners file for status.
```

Comment:
We have OWNERS file in the root dir and separate OWNERS files for each KubeCon. It'd be good to make it clear which OWNERS file the students should check.

Reply:
"Check the Owners file ..." s/Owners/OWNERS

Reply:
That makes sense - added some clarity.